### DISCUSSION TITLE
1.2.2 stable

### DISCUSSION THREAD

**Discussion by trueromanus** [anilibria/anilibria-winmaclinux#104] [Анонсы]
- Исправлена ошибка изменения громкости с помощью клавиатуры
- Убран лишний переключатель рядом с полем Тема
- Загрузка тем на странице "Менеджер Тем" > "Доступные для скачивания" после изменения визуальной темы
- Кнопка выбора цвета заменена на панельку с предпросмотром цвета
- Добавлена кнопка возврата наверх на странице "Менеджер Тем" > "Редактор темы"
- Добавлена настройка "Размер буфера" на странице "Видеоплеер"
- Добавлена кастомизация цветов - Цвет рамки постера релиза, Цвет не активного текста для кнопки, Цвет акцента в материал элементах
- Компромиссный вариант для скрытия пунктов скоростей и качеств на странице Онлайн плеер
- Поиск поля на странице "Менеджер Тем" > "Редактор темы"
- После сохранения новой темы в редакторе тем она сразу применяется если была выбрана
- Исправлена ошибка дублирования темы после редактирования темы
- Исправлена ошибка зависания кнопки Переопределить поле
- Поиск на странице Моя Анилибрия и возможность искать по жанрам, годам и т.п. прямо в строке поиска
- Добавлена кастомизация иконок из главного меню и кастомного тулбара

<hr /><em>This discussion was created from the release <a href='https://github.com/anilibria/anilibria-winmaclinux/releases/tag/1.2.2'>1.2.2 stable</a>.</em>

**Comment by liminalit** [comment]
При попытке скачать торрент либо ничего не происходит, либо выдает такую ошибку.
https://yapx.ru/v/VJA3L

**Reply by trueromanus** [reply]
Возможно файл не скачался полностью, или проблема в торрент клиенте. Скопируйте сюда (приложите) пожалуйста торрент файл по этому пути, я проверю.
